### DISCUSSION TITLE
Small update

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#688] [Announcements]
Because of some reason(暑假作业), I cannot contribute to this project in the next 10 days. Hopefully swx gets less lazy. I don’t use xmoj anyways, so why should I care if you don’t get updates.

**Comment by boomzero** [comment]
I suppose I have to @XMOJ-Script-dev/lazy-product-managers and @langningchen

**Comment by boomzero** [comment]
Also I am proud to announce that xmoj-script is somehow more stable than stupid exlg which has far more users

[a comment by an automated account is not shown]

**Reply by boomzero** [reply]
As mentioned, I do not care.

**Comment by boomzero** [comment]
I have set the status of all repos to ignore

**Comment by langningchen** [comment]
I've found an excellent project, it's called exhloj, but it's not open source......

**Comment by langningchen** [comment]
But I managed to stole it's source, although it is compiled, I can sill found the structure of code is excellent.

**Reply by boomzero** [reply]
illegal.....

**Reply by boomzero** [reply]
I am sure that you can not decompile without explicit permission.
